polkadot/node/core/pvf/src/testing.rs

```diff
@@ -71,22 +104,37 @@ pub fn get_and_check_worker_paths() -> (PathBuf, PathBuf) {
 		let mut execute_worker_path = workers_path.clone();
 		execute_worker_path.push(EXECUTE_BINARY_NAME);
 
+		let mut build_prep = false;
+		let mut build_exec = false;
+
 		// Check that the workers are valid.
-		if !prepare_worker_path.is_executable() || !execute_worker_path.is_executable() {
-			panic!("ERROR: Workers do not exist or are not executable. Workers directory: {:?}", workers_path);
+
+		if !prepare_worker_path.is_executable() {
+			eprintln!("WARNING: Prepare worker does not exist or is not executable. Workers directory: {:?}", workers_path);
```

Comment:
Not sure these warnings have any value, if we build the workers any way.

Reply:
Still informative for some cases, let's keep them.

Reply:
Which cases do you mean? Like if they get rebuilt and are still not executable? Then this could be a panic, including the command to build them manually.

Reply:
Oh, I meant it could be a bit frustrating if you know you have the workers in place but a build still happens. It'd be nice to clarify that the worker is stale. Even for people who are aware of that, it doesn't hurt to see those messages.

Reply:
Hmm, when I try it I don't see the messages unless `cargo test -- --nocapture` is passed. Even then, there is so much output that they are easy to miss.

If we can figure out a way to *always* print the messages, that would be ideal, and I'd also try to make them stand out:

```
========================
WARNING:

Blah blah workers not available, rebuilding
========================
```

Reply:
Why clutter test output with this? This doesn't bring any benefit for the running tests.

Reply:
These are merely informational, nothing to take action for, it's ok that people miss them I guess.

Reply:
It's possible for the workers to be rebuilt if the code changed but the version didn't change. In that case the warnings wouldn't be displayed. So these warnings, even if you pass `-- --nocapture`, are not even consistent. I'd personally vote for dropping them, but I suppose they don't hurt either.